### DISCUSSION TITLE
希望下个版本固件中Portainer默认汉化

### DISCUSSION THREAD

**Discussion by AshakurasA** [coolsnowwolf/lede#9593] [R68S / 电犀牛专区]
如题所述，希望Lean下次更新能把Portainer汉化一下![J4M4_ (MR4P{JI $6JRCMFR](https://user-images.githubusercontent.com/106734337/174093017-011f1dfa-33f8-4f22-9456-9911eda83710.png)

**Comment by misakamik0t0** [comment]
兄弟，为啥我的界面啥都没有
![Uploading Q9%Y57A0AT@1(ZWUREUANOA.png…]()

**Comment by misakamik0t0** [comment]
![Q9%Y57A0AT@1(ZWUREUANOA](https://user-images.githubusercontent.com/53134877/174119396-d0b37cda-742c-4327-b363-83082bd7acae.png)
兄弟为啥我界面啥都没有啊

**Reply by AshakurasA** [reply]
你是保存配置升级的固件吧

**Reply by misakamik0t0** [reply]
是的。。要线刷是吗。。

**Reply by misakamik0t0** [reply]
> 你是保存配置升级的固件吧

是的。。要线刷是吗。。

**Reply by AshakurasA** [reply]
网页升级的时候 保存配置 这个别勾就行了
